src/cli/sql.rs

```diff
@@ -268,7 +279,9 @@ fn process(pretty: bool, json: bool, res: surrealdb::Result<Response>) -> Result
 		(false, true) => output
 			.iter()
 			.enumerate()
-			.map(|(i, v)| format!("-- Query {:?}\n{v:#}", i + 1))
+			.map(|(i, v)| {
+				format!("-- Query {:?} (took {:?})\n{v:#}", i + 1, stats[i].execution_time)
```

Comment:
Shouldn't we make it clear what "took" here is referring to? In future we will be adding more stats and we may need to show those stats here as well. Maybe something like `(took {:?}  to execute)`? Or we can just say `(execution time: {:?})`. What do you think @kearfy?

Reply:
Yeah, makes sense